internal/env/util.go

```diff
@@ -0,0 +1,46 @@
+package env
+
+import (
+	"os"
+	"strconv"
+)
+
+func IsCLICloudShell() bool {
+	cliCloudShell, _ := strconv.ParseBool(os.Getenv(DevboxCLICloudShell))
+	return cliCloudShell
+}
+
+func IsDevboxCloud() bool {
+	return os.Getenv(DevboxRegion) != ""
+}
+
+func IsDevboxShellEnabled() bool {
+	inDevboxShell, _ := strconv.ParseBool(os.Getenv(DevboxShellEnabled))
+	return inDevboxShell
+}
+
+func NotTrack() bool {
```

Comment:
nit: `DoNotTrack` reads more naturally

Reply:
But the name is occupied by the env constant...

Reply:
I see. I'll create a PR to do so.

Reply:
See https://github.com/jetpack-io/devbox/pull/954